test/eradius_metrics_SUITE.erl

```diff
@@ -130,8 +130,9 @@ check_single_request(error, EradiusRequestType, RequestType, ResponseType) ->
     ok = check_metric(?SERVER_ID_FORMAT_ERROR(request, total, counter), 2).
 
 check_total_requests(good, N) ->
-    ok = check_metric(?SERVER_ID_FORMAT_GOOD_TOTAL(request, total, counter), N),
-    ok = check_metric(?SERVER_ID_FORMAT_GOOD_TOTAL(response, total, counter), N);
+    % accounting create additional +1 metrics for {?RStatus_Type, ?RStatus_Type_Start} attributes
+    ok = check_metric(?SERVER_ID_FORMAT_GOOD_TOTAL(request, total, counter), N + 1),
+    ok = check_metric(?SERVER_ID_FORMAT_GOOD_TOTAL(response, total, counter), N + 1);
```

Comment:
this is suspicious, why would the accounting request accounted twice?

Reply:
Looks like because we have `accountRequests` and `accountRequestsStart` metrics and as total of those metrics is `2`. Eg: we have `accountRequestsStart: 1` and `accountRequestsStop: 1` and also we keep all `accountRequest` and `accountRequest` will be `2` and total accounting metrics of those points will be `4`.

Reply:
Fixed